homeassistant/helpers/integration_platform.py

```diff
@@ -48,6 +48,10 @@ def _get_platform(
         )
         return None
 
+    if integration.platform_exists(platform_name) is False:
```

Comment:
The one here is called significantly more (order of magnitude more at least). If we get to `integration.get_platform` the amont of blocking I/O it will do is two or three orders of magnitude more than the `stat()` calls